client/scripts/keygen.sh

```diff
@@ -7,17 +7,22 @@ set -e
 ID_DESTINATION=$HOME/.ssh/id_rsa_inertia_deploy
 PUB_ID_DESTINATION=$HOME/.ssh/id_rsa_inertia_deploy.pub
 
-# Check if destination file already exists:
-if [ -f $ID_DESTINATION ]; then
-    if [ ! -f $PUB_ID_DESTINATION ]; then
+# Install openssh if ssh-keygen is not available
+if hash ssh-keygen 2>/dev/null; then
```

Comment:
What does `hash` do here?

Reply:
I'm not too sure, but @chadlagore used it [here](https://github.com/ubclaunchpad/inertia/blob/master/client/scripts/docker.sh#L41) to check for `curl`

Reply:
Seems like it's a shell built-in the caches filenames in `$PATH`:
https://unix.stackexchange.com/questions/86012/what-is-the-purpose-of-the-hash-command

Reply:
nifty!